tests/unit/karma.conf.js

```diff
@@ -26,6 +26,7 @@ module.exports = function (config) {
             'media/js/base/mozilla-run.js',
             'media/js/base/core-datalayer-page-id.js',
             'media/js/base/core-datalayer.js',
+            'media/js/base/datalayer-begincheckout.es6.js',
```

Comment:
Nit: no need to add this here when you're importing the module directly into the test file